### ISSUE TITLE
Add Plotly in Powered by Dask section in homepage

### ISSUE THREAD

**Issue by rpanai** [dask/dask.github.io#36]
As in title Plotly with Dash Enterprise is using Dask [https://plotly.com/dash/gpu-dask-acceleration/](https://plotly.com/dash/gpu-dask-acceleration/) I think this satisfies  _use Dask to power components of their infrastructure_ in [dask.org](https://dask.org/)

**Comment by martindurant** [comment]
Note the typo on the website under Dask for CPU Parallelization:
"Dask Enterprise ships with out-of-the-box support for Dask"

**Comment by jrbourbeau** [comment]
@rpanai I moved this issue from dask/dask to here as this is where the source files for dask.org live

> This list so far is about OSS projects.  I wouldn't add Dash Enterprise to this list

I agree that I wouldn't want to add a project that's not free and open source to the current list. So Dash Enterprise wouldn't qualify as free and open source, but IIUC [plotly](https://github.com/plotly/plotly.py) would. @rpanai do you know if plotly by itself uses Dask?

**Comment by rpanai** [comment]
@jrbourbeau Plotly doesn't use dask. It was not clear to me that it was restricted to free and open source project only. I would consider to specify it, or add a section for not OSS. I don't see the harm highlighting that Dask is used by companies to scale their architecture.

**Comment by mrocklin** [comment]
Yeah, I think that it would be great to have a section that highlights all
of the companies that use Dask.  Dash Enterprise by Plotly would
definitely be a good candidate there.  I'll raise an issue on the community
issue tracker to get buy-in, and then I'll tweet out asking publicly for
entries.

**Comment by jrbourbeau** [comment]
> It was not clear to me that it was restricted to free and open source project only

I agree we could do a better job of structuring / documenting the landing page. There have been a few related issues in the community issue tracker on the topic of acknowledging companies that use and/or support Dask (xref https://github.com/dask/community/issues/44, https://github.com/dask/community/issues/32)

>  I don't see the harm highlighting that Dask is used by companies to scale their architecture

There's certainly value in that and I look forward to engaging on the issue @mrocklin raises

**Comment by mrocklin** [comment]
Actually James, do you want to raise that issue?

**Comment by jrbourbeau** [comment]
Yeah, happy to

**Comment by jrbourbeau** [comment]
Here's the issue in the Dask community tracker https://github.com/dask/community/issues/102
